Run/navstart.ps1

```diff
@@ -251,13 +257,15 @@ if ($runningGenericImage -or $runningSpecificImage -or $buildingImage) {
     $CustomConfig.Save($CustomConfigFile)
 }
 
-if ($runningGenericImage -or $runningSpecificImage) {
+if ($runningGenericImage -or $runningSpecificImage -or $restartingInstance) {
```

Comment:
not really connected with the problem above but stopping the local instance always makes sense if the databaseserver isn't localhost